### PR TITLE
fix(telephony.service.voicemail): replace aapi call with autocomplete

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-22218
| License          | BSD 3-Clause

## Description

2API timeout due to multiple calls and huge amount of data to fetch/return
This will also improve performances for large account as we avoid building a huge amount of options in a select

## Related 

- [x] TRANS-29069

## TODOs
- [x] Check that 2API is no longer used to remove or replace left calls

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
-                        class="form-control"
-                        required
+                        class="oui-input"
                         data-ng-model="DefaultVoicemailCtrl.options.defaultVoicemail"
-                        data-ng-change="DefaultVoicemailCtrl.success = false"
-                        data-ng-options="number.serviceName as DefaultVoicemailCtrl.formatNumber(number) for number in DefaultVoicemailCtrl.numbers | orderBy: ['description', 'label', 'serviceName']"
-                    >
-                    </select>
+                        type="text"
+                        oui-autocomplete="DefaultVoicemailCtrl.numbers"
```
Comment (radireddy) [inline packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html:28]: 👍

File: packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html [hunk 2]
```diff
@@ -19,16 +19,16 @@
                         data-translate="telephony_line_answer_default_voicemail_label"
                     >
                     </label>
-                    <select
+                    <input
```
Comment (radireddy) [inline packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html:22]: Do you think a help text like "enter first few numbers to search" will help users?

**Review by radireddy — APPROVED** [review]

**Review by frenautvh — DISMISSED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html (hunk 2 above)
Comment (marie-j) [reply packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html:22]: You're right, a placeholder could be nice, I'll add it

**Review by JeremyDec — APPROVED** [review]

**Comment by marie-j** [comment]
Added a placeholder as suggested by @radireddy

**Review by cbourgois — APPROVED** [review]

**Review by Heiiji — DISMISSED** [review]

**Comment by ovh-cds** [comment]
CDS Report translator-push#11714.0 ■   
* 
  * push ■

**Comment by ovh-cds** [comment]
CDS Report translator-push#11714.0 ■   
* 
  * push ■

**Review by antleblanc — APPROVED** [review]

**Comment by antleblanc** [comment]
@marie-j Could you please resolve existing conflicts?
Did you have time to check the 2API side as well?

Thanks!

**Comment by marie-j** [comment]
Hello @antleblanc ,

Conflicts are now fixed. I didn't find any other use of this 2API across manager repository

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**